examples/caching_and_replay/README.md

```diff
@@ -0,0 +1,42 @@
+# Caching and Replaying Schelling
```

Comment:
Should be "Schelling Model with Caching and Replay".

Reply:
changed